radix-engine/src/vm/wasm_runtime/scrypto_runtime.rs

```diff
@@ -588,10 +591,18 @@ where
             scrypto_decode(&pub_keys_and_msgs)
                 .map_err(WasmRuntimeError::InvalidBlsPublicKeyOrMessage)?;
 
-        let result = self
-            .api
-            .bls12381_v1_aggregate_verify(&pub_keys_and_msgs, &signature)?;
-        Ok(result)
+        if pub_keys_and_msgs.is_empty() {
+            return Err(InvokeError::SelfError(WasmRuntimeError::InputDataEmpty));
+        }
```

Comment:
Is this a breaking change?

Reply:
The check was in the `api.bls12381_v1_aggregate_verify()` call, the logic of which has now moved into this function.